pyproject.toml

```diff
@@ -53,7 +53,8 @@ msal = "^1.16.0"
 python-jose = {extras = ["cryptography"], version = "^3.3.0"}
 aiojobs = "^1.0.0"
 aiohttp-pydantic = "^1.12.1"
-virtool-core = "^2.26.2"
+virtool-core = "^2.27.1"
+setuptools = "^65.3.0"
```

Comment:
Where did this come from?

Reply:
I was having an error only on my local machine where pkg_resources from setuptools was not found, so I was unable to run any tests. This must have been added accidentally as I was trying to solve the issue.